app/docker-compose.yml

```diff
@@ -39,6 +39,6 @@ services:
     volumes:
       - "./data/media/uploads/:/uploads/"
     ports:
-      - 5000:5000
+      - 5001:5000
```

Comment:
Please don't commit this :)

Reply:
AirPlay uses 5000 so I had to change it for myself. I guess it makes more sense for me to just turn AirPlay off than to commit this and confuse people.